### PR TITLE
Fix vars completion in task files

### PR DESCRIPTION
**Pull request by priyamsahoo**
The PR:
1. Fixes the issue of ls being stuck when asked for vars completion in a task file (Resolves: #587).
2. Fixes extra indentation issue when module completion is resolved.

### CODE REVIEW

**Comment by ajinkyau** [comment]
@priyamsahoo do we have tests to cover these scenarios?

File: src/providers/completionProvider.ts [hunk 1]
```diff
@@ -51,13 +52,16 @@ const priorityMap = {
   choice: 2,
 };
 
-let DUMMY_MAPPING_CHARACTERS;
+let DUMMY_MAPPING_CHARACTERS: string;
+let IS_PLAYBOOK: boolean;
```
Comment (ajinkyau) [inline src/providers/completionProvider.ts:56]: Any particular reason we're using this naming convention?

Comment (priyamsahoo) [reply src/providers/completionProvider.ts:56]: The initial assumption was that these vars would be constants, but based on the logic, these vars have to be changed. I will update the names. Thanks!

**Comment by priyamsahoo** [comment]
> @priyamsahoo do we have tests to cover these scenarios?

We do have tests for the doCompletionResolve as a whole. But that needs to be changed. I will convert this PR into draft and open it again once I update the existing tests.

File: src/providers/completionProvider.ts (hunk 1 above)
Comment (ajinkyau) [reply src/providers/completionProvider.ts:56]: @priyamsahoo would you be updating the variable name?

**Review by ajinkyau — CHANGES_REQUESTED** [review]

**Comment by priyamsahoo** [comment]
I initially forgot to change it. Updated now!

**Review by ajinkyau — APPROVED** [review]

**Review by ssbarnea — APPROVED** [review]

**Pull request merged**